backend/src/models/task.ts

```diff
@@ -0,0 +1,30 @@
+import mongoose from 'mongoose'
+const { Schema } = mongoose
+
+export interface ITask {
```

Comment:
We have more information about the task, such as "label", "due date", "priority", if those could be added!